<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>802</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans Mono</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Secure Outlook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;DejaVu Sans Mono&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QPushButton" name="composebtn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>50</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;Cambria&quot;;
background-color:rgb(7, 119, 197);
border-radius:20px;
</string>
    </property>
    <property name="text">
     <string>Compose</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>Inbox</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Text_9">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Created by:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Anthony Amgad Fayek : 19P9880&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Ragui Chawkat Naguib : 20P1355&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>Hover over me!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshbtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;Cambria&quot;;
background-color:rgb(7, 119, 197);
border-radius:50%;
border: none;
</string>
    </property>
    <property name="text">
     <string>â†º</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>311</width>
      <height>751</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fromLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Text_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 12pt;</string>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Text_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 12pt;</string>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="toLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Text_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 12pt;</string>
    </property>
    <property name="text">
     <string>Subject:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="subjectLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Text_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 12pt;</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="attCom">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 232, 247);</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Pick a Node to Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openAttbtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;Cambria&quot;;
background-color:rgb(7, 119, 197);
border-radius:20px;
</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text_10">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(7, 119, 197);
font: 75 12pt;</string>
    </property>
    <property name="text">
     <string>Attachments:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>listWidget</zorder>
   <zorder>composebtn</zorder>
   <zorder>Text_2</zorder>
   <zorder>Text_9</zorder>
   <zorder>refreshbtn</zorder>
   <zorder>fromLabel</zorder>
   <zorder>Text_3</zorder>
   <zorder>Text_4</zorder>
   <zorder>toLabel</zorder>
   <zorder>Text_7</zorder>
   <zorder>subjectLabel</zorder>
   <zorder>Text_8</zorder>
   <zorder>dateLabel</zorder>
   <zorder>attCom</zorder>
   <zorder>openAttbtn</zorder>
   <zorder>Text_10</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
